apps/vaults/hooks/useSolverCowswap.ts

```diff
@@ -245,13 +266,13 @@ export function useSolverCowswap(): TSolverContext {
 				const {isSuccessful, error} = await checkOrderStatus(orderUID, quote.validTo);
 				if (error) {
 					console.error(error);
-					return ({isSuccessful: false, error});
+					return ({isSuccessful: false, error: error as BaseError || ''});
```

Comment:
I think we might need a type guard for this, not sure we should be casting to `BaseError` as that might not be the case. Wdyt?